### DISCUSSION TITLE
Created Slack channels for non-English users - #mirth_auf_deutsch #mirth_en_espanol

### DISCUSSION THREAD

**Discussion by jonbartels** [nextgenhealthcare/connect#6012] [General]
A few users in Slack had the idea to create channels for users who did not speak English as a first language.

Join Slack at https://github.com/nextgenhealthcare/connect/blob/development/README.md#1-useful-links

#mirth_auf_deutsch 
#mirth_en_espanol

I hope this helps welcome new users to the community.

**Comment by rjkroll** [comment]
What is the link for the English channel?   Everytime I try to click on the one that you can find posted around, it doesn't work for me for some reason.

**Reply by pacmano1** [reply]
https://github.com/nextgenhealthcare/connect/wiki it is in the second paragraph.

**Reply by tonygermano** [reply]
It's also in the first section of the README file under Useful Links.

https://github.com/nextgenhealthcare/connect/blob/9ae628546bfc453c7e7ced5237adde8c52b75b3e/README.md?plain=1#L22

**Reply by jonbartels** [reply]
[English Channel](https://en.wikipedia.org/wiki/English_Channel)

**Reply by pacmano1** [reply]
> [English Channel](https://en.wikipedia.org/wiki/English_Channel)

Clever
